benchmarks/tpcc/src/main/java/com/google/cloud/pgadapter/tpcc/BenchmarkApplication.java

```diff
@@ -118,10 +118,7 @@ public void run(String... args) throws Exception {
       }
 
       if (tpccConfiguration.isRunBenchmark()
-          && (tpccConfiguration.getBenchmarkRunner().equals(TpccConfiguration.PGADAPTER_JDBC_RUNNER)
-              || tpccConfiguration
-                  .getBenchmarkRunner()
-                  .equals(TpccConfiguration.SPANNER_JDBC_RUNNER))) {
+          && (TpccConfiguration.RUNNERS.contains(tpccConfiguration.getBenchmarkRunner()))) {
```

Comment:
nit: should we show a clear error message if an invalid benchmark runner name has been given? Now we just skip running any benchmarks.

Reply:
If you look at `else`, we actually have the following: 

```
LOG.error("Unknown benchmark runner option: " + tpccConfiguration.getBenchmarkRunner());
```

Updated to throw an exception.